src/cpu/aarch64/jit_generator.hpp

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
 * Copyright 2016-2020 Intel Corporation
```

Comment:
```suggestion
* Copyright 2016-2021 Intel Corporation
```

Reply:
Thank you for the suggestion. I'll fix it.